File: TestPodToday/TestPodToday/ViewController.m
//
//  ViewController.m
//  TestPodToday
//
//  Created by 张如泉 on 16/3/17.
//  Copyright © 2016年 quange. All rights reserved.
//

#import "ViewController.h"
#import "AFNetworking.h"
#import "UALogger.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    AFNetworkReachabilityManager *afNetworkReachabilityManager = [AFNetworkReachabilityManager sharedManager];
    [afNetworkReachabilityManager startMonitoring];  //开启网络监视器；
    
    [afNetworkReachabilityManager setReachabilityStatusChangeBlock:^(AFNetworkReachabilityStatus status) {
        
        switch (status) {
            case AFNetworkReachabilityStatusNotReachable:{
                
                break;
            }
            case AFNetworkReachabilityStatusReachableViaWiFi:{
                
                
                break;
            }
            case AFNetworkReachabilityStatusReachableViaWWAN:{
                
                break;
            }
            default:
                break;
        }
        
        UALog(@"网络状态数字返回：%i", status);
        UALog(@"网络状态返回: %@", AFStringFromNetworkReachabilityStatus(status));
        
    }];

}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Girls/Girls/ViewController.swift
//
//  ViewController.swift
//  Girls
//
//  Created by 张如泉 on 16/3/7.
//  Copyright © 2016年 quange. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    

    @IBOutlet weak var advImageview: UIImageView!
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        self.navigationController?.navigationBarHidden = true;
        
        dispatch_after(dispatch_time(DISPATCH_TIME_NOW, Int64(2 * Double(NSEC_PER_SEC))), dispatch_get_main_queue()) {
            self.gotoAppHome()
        }
        self.advImageview.userInteractionEnabled = true
        self.advImageview.addGestureRecognizer(UITapGestureRecognizer(target: self, action: "gotoAdvDetail:"))
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    

    @IBAction func testCrash(sender: UIButton) {
       self.gotoAppHome()
    }
    
    func gotoAppHome()
    {
        if(self.view.tag != 111)
        {
            let main = UIStoryboard(name: "Main", bundle: nil);
            let modal=main.instantiateViewControllerWithIdentifier("GHomeViewController");
            self.navigationController?.pushViewController(modal, animated: false);
        }
    }
    
    func gotoAdvDetail(sender:UITapGestureRecognizer)
    {
        
        gotoAppHome()
        self.view.tag = 111;
    }

   
}

